yt/utilities/logger.py

```diff
@@ -74,7 +74,8 @@ class DuplicateFilter(logging.Filter):
     # source
     # https://stackoverflow.com/questions/44691558/suppress-multiple-messages-with-same-content-in-python-logging-module-aka-log-co  # noqa
     def filter(self, record):
-        current_log = (record.module, record.levelno, record.msg)
+        msg = record.msg % record.args
+        current_log = (record.module, record.levelno, msg)
```

Comment:
There's probably no need to perform the formatting operation, how about this ?
```suggestion
        current_log = (record.module, record.levelno, record.msg, record.args)
```

Reply:
Sure, I like it.